Doc/library/concurrent.futures.rst

```diff
@@ -159,6 +159,15 @@ And::
    .. versionchanged:: 3.7
       Added the *initializer* and *initargs* arguments.
 
+   .. versionchanged:: 3.8
+      Default value of *max_workers* is changed to ``min(32, os.cpu_count() + 4)``.
+      This default value preserves at least 5 worker for I/O bound task.  It utilize
```

Comment:
Not a native English speaker but I think it should be "It utilizes" and "And it avoids" in the next sentence.

Reply:
Also, `at least 5 workers`.